### DISCUSSION TITLE
Github W-9 for billing

### DISCUSSION THREAD

**Discussion by bejankowski1** [community/community#73242] [New to GitHub]
Does anyone have a copy of the W-9 for Github? I need to to make them a vendor for my company.

Thank you!!

**Comment by b8zeek** [comment]
I guess this is what you're looking for:

![Screenshot 2023-11-03 at 23 02 28](https://github.com/community/community/assets/24759563/c4d87bae-b7de-4895-82d5-cd32354a7f31)

Take care! 🍺 

🍀

**Reply by Dominic-Blake** [reply]
this doesn't exactly explain how to get it.
